code/modules/hydroponics/seed_storage.dm

```diff
@@ -59,6 +59,7 @@
 		/obj/item/seeds/chiliseed = 3,
 		/obj/item/seeds/chickpeas = 3,
 		/obj/item/seeds/clam = 15,
+		/obj/item/seeds/cocaseed = 3,
```

Comment:
the coca plant should probably not be available to the general public, considering it's a narcotic.

Reply:
See above, this won't be available after all.